.gitignore

```diff
@@ -53,3 +53,4 @@ config/database.yml
 vcr.log
 
 .byebug_history
+config/application.yml
```

Comment:
@johnnymo87 yeah, I think so, according to method `gmap_key_value_for_url` defined in **application_helper.rb**.